Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelDeletionProcessTests
+{
+    [Fact]
+    public void Should_cancel_deletion_process_during_grace_period()
```

Comment:
We don't use `Should` in test names. Tests are supposed to be documentation. And if a documentation for a product says that it **should** do something, it sounds like "I wish it would, but sometimes it doesn't".